Assets/MixedRealityToolkit/Graph/GraphResponse.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace Microsoft.MixedReality.Toolkit.Graph
+{
+    /// <summary>
+    /// Class that represents a Graph response.
+    /// </summary>
+    public class GraphResponse
```

Comment:
could this be a struct?

Reply:
This one can't because it has virtual methods.